packages/kbn-i18n/src/loader.ts

```diff
@@ -54,8 +53,7 @@ function getLocaleFromFileName(fullFileName: string) {
  * @returns
  */
 async function loadFile(pathToFile: string): Promise<TranslationInput> {
-  // doing this at the moment because fs is mocked in a lot of places where this would otherwise fail
-  return JSON.parse(await promisify(fs.readFile)(pathToFile, 'utf8'));
+  return JSON.parse(await readFile(pathToFile, 'utf8'));
```

Comment:
That's probably just a personal preference, but I usually like keeping `require` strictly for javascript import tbh even if  I think it would technically work using it there.

The arguments I could find to not use `require`:
- consistency, we're usually always importing "static" files via `readFile` and not via node imports (e.g. the plugin manifests for example)
- we have an ESlint rule to avoid dynamic requires
- I'm not sure the "perma caching" without control is really an upside (even if with our current usages of i18n I don't see it being an issue)

Reply:
Fair enough... approving 😇